modules/react/toast/stories/stories_visualTesting.tsx

```diff
@@ -51,3 +51,21 @@ export const ToastStates = () => (
     </ComponentStatesTable>
   </StaticStates>
 );
+
+export const ToastStatesLeftToRight = () => {
+  return (
+    <>
+      <h2>Left-To-Right Toast</h2>
+      <ToastStates />
+    </>
+  );
+};
+
+export const ToastStatesRightToLeft = () => {
+  return (
+    <>
+      <h2>Right-To-Left Toast</h2>
+      <ToastStates direction={ContentDirection.RTL} />
+    </>
+  );
+};
```

Comment:
This is great! Thanks for adding the regression tests! Now we'll be sure not to miss this. 👏🏻